src/Console/ModelsCommand.php

```diff
@@ -82,8 +82,8 @@ public function handle()
         $filename = $this->option('filename');
         $this->write = $this->option('write');
         $this->dirs = array_merge(
-            $this->laravel['config']->get('ide-helper.model_locations'),
-            $this->option('dir')
+            (array) $this->laravel['config']->get('ide-helper.model_locations'),
```

Comment:
This is the same as https://github.com/barryvdh/laravel-ide-helper/pull/835#discussion_r363038544
- don't cast, but provide a default value

Reply:
The reason for this not being an array is because in config it was assigned with a different type. Casting to an array would only hide the fact there is a misconfiguration.
Perhaps it would be smart to introducing a config value object/DTO that is made available from the DI. On creation it should validate the config, this to avoid scattered configuration checks throughout the code.